Ginger/GingerCoreNET/GeneralLib/General.cs

```diff
@@ -308,16 +308,14 @@ public static string CheckDataSource(string DataSourceVE, ObservableList<DataSou
 
             if (DataSource.DSType == DataSourceBase.eDSType.MSAccess)
             {
-                DataSource.FileFullPath = amdocs.ginger.GingerCoreNET.WorkSpace.Instance.SolutionRepository.ConvertSolutionRelativePath(DataSource.FileFullPath);
-
-                DataSource.Init(DataSource.FileFullPath);
-                ObservableList<DataSourceTable> dsTables = DataSource.GetTablesList();
-                foreach (DataSourceTable dst in dsTables)
+                foreach (DataSourceTable dst in DataSource.DSTableList)
```

Comment:
this might resolve the error in analyzer but did you test Data source functionality after this change ?
Because DSTableList is initialized to empty list in DataSource class.

Also what is the issue with  GetTableList method? if it is not working or not needed let's remove it completely